src/components/media-types/index.js

```diff
@@ -36,6 +37,10 @@ export const renderMediaType = ({
       return (
         <VectorComponent {...metadata} src={url} interactive={interactive} />
       )
+    /* HTML */
+    case MIMETYPE.HTML:
+      url = preview ? uri : `${CLOUDFLARE}${path}`
+      return <HTMLComponent src={url} interactive={interactive} preview={preview} />
```

Comment:
check this: https://github.com/hicetnunc2000/hicetnunc/blob/main/src/components/media-types/index.js#L37

I've injected some properties on the SVG component that allows me to inject 2 props, owner and viewer in case you want the content of your html hashed to whoever is seeing your creation.

Reply:
(Only injected on the objkt detail page)

Reply:
Oh nice! The hashing is a really great idea. Went ahead and added it: https://github.com/hicetnunc2000/hicetnunc/pull/189/commits/2042b1dfaa1919b9403f706bda4236fa14759b88

Reply:
Ah we should probably add the CSP meta tag to SVG content too, no? Before there was a `sandbox-svg.html` file but where would be the place now?

Reply:
Sorry I see what changed now. Still not sure of best way to add the meta tag to the SVG iframe, but maybe that can be a separate issue to this?

Hope its ok I made a small change to `VectorComponent` because it wasn't displaying properly in preview mode with the author and viewer query params in the src since its a data URI at that point (unless I'm misunderstanding something?):
https://github.com/pichiste/hicetnunc/commit/a02e60b69bb126683faf9809633589e3fcafd023

Reply:
Resolving for now but let me know if anything needs to change, thanks!!

Reply:
The hash wasn't available in preview mode because I wasn't putting in a url to a file. But if this fixes it then great! I'm gonna give this a test tonight. Can you drop a html template on the templates folder now on main? Just so I can use to test the file upload

Reply:
Cool, just added an html template (https://github.com/hicetnunc2000/hicetnunc/pull/189/commits/9c5e9b4bca18b0c2c9b2199c42bdd81fa37920a2)!

A heads up, both svg and html templates reference the query param `creator` but it's being set as `author` in `VectorComponent` and `HTMLComponent`. Left it that way since I didn't know if it's being used elsewhere.